_dashboard/DotNetPerfMonitor/public/app_config.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "DotNetPerfMonitor",
+  "version": "1.0.0",
+  "description": "DotNetPerfMonitor....",
+  "components": [
+    { "uid": "nuget", "name": "NuGet" },
+    { "uid": "msbuild", "name": "MSBuild" },
+    { "uid": "fsharp", "name": "F#" },
+    { "uid": "csharp", "name": "C#" },
+    { "uid": "aspnet", "name": "ASP.NET" },
+    { "uid": "xunit", "name": "Xunit" }
+  ],
+  "data": {
+    "nuget": {
+      "data": "https://raw.githubusercontent.com/G-Research/DotNetPerfMonitor/main/data/nuget.csv",
+      "benchmarks": [
+        {
+          "name": "LargeAppCPM64-142722b",
+          "color": "rgb(144, 12, 63)"
+        },
+        {
+          "name": "LargeAppCPM64-nostaticgraph-142722b",
+          "color": "rgb(248, 222, 34)"
+        },
+        {
+          "name": "LargeAppCPM64-142722b",
+          "color": "rgb(117, 194, 246)"
+        },
+        {
+          "name": "LargeAppCPM-142722b",
+          "color": "rgb(255, 14, 64)"
+        },
+        {
+          "name": "OrchardCore-5dbd92c",
+          "color": "rgb(255, 184, 76)"
+        },
+        {
+          "name": "Orleans-eda972a",
+          "color": "rgb(44, 211, 225)"
+        },
+        {
+          "name": "NuGetClient-win-d76a117",
+          "color": "rgb(255, 99, 132)"
+        }
```

Comment:
It is ok, for now - but think how to assign colours programmatically so we could add more benchmarks to the `nuget.csv` file without need to update the `app_coinfig.json` file.

Reply:
Yeah! I created a composable `useColorGenerator` to randomly a RGB color.

But it is so dynamic, anytime the chart is loaded, the colors change.

That's why I added colors in app_config ......

If this is not OK I can simply switch back to color generator we used to have